### PR TITLE
Update jet configs; minor doc update

### PR DESCRIPTION
**Pull request by ulmononian**
- Add a `mirror.yaml` for Jet
- Add `libxpm` as an an external in Jet packages.yaml (breaks UE install w/out it).
- Update Jet prereq. load section in Platforms chapter
- Add `spack install --source --verbose` to Maintainers chapter

### CODE REVIEW

**Comment by climbfuji** [comment]
This needs to go into the release branch, I'll make the change.

File: doc/source/Platforms.rst [hunk 1]
```diff
@@ -393,8 +393,17 @@ Note that on Hera, a dedicated node exists for ``ecflow`` server jobs (``hecflow
...
+
+   module purge
+   module use /lfs4/HFIP/hfv3gfs/spack-stack/modulefiles
+   module load miniconda/3.9.12
+   module load ecflow/5.5.3
+   module use /lfs4/HFIP/hfv3gfs/role.epic/modulefiles
+   module load mysql/8.0.31
+
 
 **WORK IN PROGRESS**
```
Comment (climbfuji) [inline doc/source/Platforms.rst]: Do we still need the work in progress? I guess it's ok and we add the information for loading the spack-stack-1.3.0 unified-env for all platforms at once later.

Comment (climbfuji) [reply doc/source/Platforms.rst]: Same goes for the table at the top of the page.

**Review by climbfuji — APPROVED** [review]

File: doc/source/Platforms.rst (hunk 1 above)
Comment (ulmononian) [reply doc/source/Platforms.rst]: that is why i left it...will remove when we add the spack-stack-1.3.0 unified-env as you said.

**Comment by climbfuji** [comment]
You'll need to resolve the conflicts in `doc/source/Platforms.rst` before it can be merged.

**Comment by ulmononian** [comment]
> You'll need to resolve the conflicts in `doc/source/Platforms.rst` before it can be merged.

am i missing something? i updated to the release version of PreconfiguredSites.rst and added the jet change. idk how that is going to be resolved...

**Comment by climbfuji** [comment]
I'll fix it tomorrow and either push to your branch our send you a PR to update your branch.

**Comment by ulmononian** [comment]
> I'll fix it tomorrow and either push to your branch our send you a PR to update your branch.

thanks for doing that, @climbfuji.

**Pull request merged**